sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -31,6 +33,24 @@ def streaming_test():
 def stream_compressed_header_error():
     yield b'test'
 
+def stream_compressed_no_header():
+    try:
+        with gzip.open('test.tar.gz', 'wb') as f:
+            f.write(b"test")
+
+        with open(os.path.join(os.path.abspath('test.tar.gz')), "rb") as fd:
+            yield fd.read()
+
+        os.remove("test.tar.gz")
+    except:
+        with gzip.open('test.tar.gz', 'wb') as f:
+            f.write(b"test")
+
+        with open(os.path.join(os.path.abspath('test.tar.gz')), "rb") as fd:
+            yield fd.read()
+            
+        os.remove("test.tar.gz")
+
```

Comment:
yay! it looks like this fixed the failing tests (other test failures are unrelated). One small thing - we don't need the try/except code anymore since we only added that before when we weren't sure what the expected path to the file was. We can remove the duplicate code here and just include lines L38-L44